02-peer-protocol.md

```diff
@@ -192,6 +192,10 @@ know this node will accept `funding_satoshis` greater than or equal to 2^24.
 Since it's broadcast in the `node_announcement` message other nodes can use it to identify peers 
 willing to accept large channel even before exchanging the `init` message with them. 
 
+Implementers are advised to provide the means to scale the number of confirmations, tweaking 
+`accept_channel.minimum_depth`, with the size of the funding amount. A rule of thumb is to 
```

Comment:
We should just drop the sentence starting with "A rule of thumb". I don't think its sufficiently conservative for many users, and its probably too conservative for others.

Reply:
I'm curious on why it wouldn't be sufficiently conservative for many users (we're also adding _largely exceeds_), but if this is too generic and can't cover most cases then I agree we should remove it.

Reply:
I think it's mainly that it's too _opinionated_. Looking at the cumulative block reward to evaluate cost is one way among several ways of evaluating your scaling scenario. Having this in the spec may be influencing too much the implementers.

IIUC what was suggested during the meeting, this advisory should only say something like:

```
Implementers are advised to provide the means to scale the number of confirmations, tweaking 
`accept_channel.minimum_depth` and `accept_channel.to_self_delay`, with the size of the funding amount.
```

And let implementers completely free of choosing what scaling strategies they offer to their users.

Reply:
Sounds good to me :+1: